docs/tools-docs.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Walkthrough
+---
+
+## In a nutshell, RMRK tools:
+
+- GET complete state of all remarks right now
+- RETRIEVE remarks from extrinsic block call
+- VALIDATE remarks
+- CREATE new remark strings ready to be sent as transactions
+
+This walkthrough of our Tools is based on the 1.0 spec. We will be releasing v2.0 docs whenever RMRK2.0 hits the shelves! For the best developer experience, we recommend developing with the most current **Mac operating system**.
```

Comment:
> we recommend developing with the most current **Mac operating system**.

We absolutely do not, lol :D Remove please. Recommend a *nix environment, that's it.